Add `Monthly Beatmapping Contest: July 2024` news post (#11753)
ppy/osu-wiki: news/2024/2024-07-16-monthly-beatmapping-contest-july-2024.md

---
layout: post
title: "Monthly Beatmapping Contest: July 2024"
date: 2024-07-16 18:15:00 +0000
---

> It might be too hot to go outside, but at least you can prove you're a top-tier mapper this Summer!

Reviewer: ```suggestion
It might be too hot to go outside, but at least you can prove you're a top-tier mapper this summer!
```

## Game modes

This month's contest is open to **osu!** and **osu!mania** mappers.

- [osu! contest listing](https://osu.ppy.sh/community/contests/216)
- [osu!mania contest listing](https://osu.ppy.sh/community/contests/217)

Next month is dedicated to osu!taiko and osu!catch.

## Timeline

- **[Beatmapping](#beatmapping) (3 weeks):** Contest is announced and beatmappers create their entries.
- **[Screening](/wiki/Contests/Monthly_Beatmapping_Contest#screening) (1.5 weeks):** Entries are evaluated by experienced beatmappers. If there aren't many entries, this will be skipped in favour of a longer judging period.
- **[Judging](/wiki/Contests/Monthly_Beatmapping_Contest#judging) (1.5 weeks):** Top entries from screening are judged in detail by a different crew of experienced beatmappers.
- **[Rewards](#rewards) (~2 months from now):** Winners are announced at the start of another MBC.

## Beatmapping

There have been over 20 Monthly Beatmapping Contests since 2020. It would be boring if every contest was the same, so we throw **themes** and **limitations** into the mix.

> These are essentially rules specific to each contest, so you can't re-use the same map each month.

Reviewer: ```suggestion
These are essentially rules specific to each contest, so you can't reuse the same map each month.
```

If you're confused about how these work, ask for clarification in the comments, or send a message to [pishifat](https://osu.ppy.sh/community/chat?sendto=3178418) (for osu!) or [RandomeLoL](https://osu.ppy.sh/community/chat?sendto=7080063) (for osu!mania).

#### osu!

Every mapper has heard something along the lines of "stream maps are so lame nowadays" and "every stream map is the same" and "Muya made the best stream map 12 years ago so what's the point".

This contest's theme: **create a stream-focused map.**

> There's no strict criteria for what classifies a stream map as *a stream map*, so this aspect will be evaluated by our judging panel! More specifically, the "theme" category in judging will encompass how stream-focused your map is and how fun/creative your streams are.

Reviewer: ```suggestion
There are no strict criteria for what classifies a stream map as *a stream map*, so this aspect will be evaluated by our judging panel! More specifically, the "theme" category in judging will encompass how stream-focused your map is and how fun/creative your streams are.
```

That said, if your plan was to submit a map that's not obviously stream-focused, you're guaranteed to lose. Make your intention clear!

#### osu!mania

> Little John was given the worst possible keyboard known to mankind. As such, Little John's keyboard only has access to 2-key rollover, and has been having trouble finding beatmaps that he's able to play. This month's restriction aims to give Little John a chance to play some charts! To make things even simpler, he only plays 4 key sets, unfortunately.

Reviewer: ```suggestion
Little John was given the worst possible keyboard known to mankind. As such, Little John's keyboard only has access to 2-key rollover, and has been having trouble finding beatmaps that he's able to play. This month's restriction aims to give Little John a chance to play some charts! To make things even simpler, he only plays 4-key sets, unfortunately.
```
